tests/test_edxapp_play.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+ROOT_DIR=${PWD}
+cd playbooks/edx-east
+output_dir="${PWD}/test_output/env-dep"
+mkdir -p ${output_dir}
+ansible-playbook -i localhost, -c local --tags edxapp_cfg edxapp.yml -e edxapp_user=`whoami` -e edxapp_app_dir=${output_dir} -e edxapp_code_dir=${output_dir} -e EDXAPP_CFG_DIR=${output_dir}
+
+root_dir=${output_dir}
+environment_deployments="."
```

Comment:
Move these two lines (8,9) into the `validate_templates` file,
then execute that file via the `Makefile`.